README.md

```diff
@@ -59,7 +59,7 @@ For the contest the focus was on 4 substrates and their corresponding validation
 | | commons_harvest__private_property_1 |
 | | commons_harvest__open_disable_zapping_0 |
 | | commons_harvest__open_disable_zapping_1 |
-
+| | commons_harvest__open_abundance |
```

Comment:
I think the last 3 entries in the readme (so not just @chris-p-bacon1 changes) need to be updated